### PR TITLE
Issue #450 - Fix doc and image links

### LINKED ISSUE #450: Bug🐛: missing externalLink directives
### Describe the bug.

There are many dozens of markdown pages in this repo that are not correctly using the `{:target="_blank"}{: .externalLink}` directive.

The below unix pipeline finds `https://` links in the markdown files that DO NOT have the externalLink directive. (I try to exclude false positives with the `-v` option, but don't filter all of the false positives)
 
```
git clone git@github.com:open-horizon/open-horizon.github.io.git
cd open-horizon.github.io/docs

find . | grep md | xargs grep -r https | grep -v "externalLink\|url\|export\|wget\|bluehorizon\|oc login\|api\|apt" | cut -d: -f1 | sort | uniq
```

Here are the markdown files that I think need to corrected:
```
./admin/backup_recovery.md
./admin/multi_tenancy.md
./anax/docs/all-in-1-setup.md
./anax/docs/attributes.md
./cli/cloudctl_oc_cli.md
./developing/best_practices.md
./developing/container_registry.md
./developing/developing_secrets.md
./developing/examples.md
./developing/model_management_details.md
./developing/secrets_details.md
./developing/service_containers.md
./fdo/README-FDO1.1.md
./getting_started/known_issues.md
./hub/lfedge_mgmt_hub.md
./installing/advanced_man_install.md
./installing/edge_cluster_agent.md
./installing/preparing_edge_cluster.md
./kubearmor-integration/docs/README.md
./mgmt-hub/docs/index.md
./user_management/security_privacy.md
./using_edge_services/cpu_load_example.md
```

### Describe the steps to reproduce the behavior

_No response_

### Expected behavior.

_No response_

### Screenshots.

_No response_

### Operating environment

all

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by johnwalicki**
# Pull Request Template

## Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes #450 
Fixes $448

- I believe this will be the last PR related to externalLinks
- Resolves issues posted in https://github.com/open-horizon/open-horizon.github.io/issues/450#issuecomment-1446573765

## Type of change

- [x] This change requires a documentation update

## How Has This Been Tested?

Visit:

-  https://open-horizon.github.io/docs/developing/container_deployment_options/ (broken link)
-  https://open-horizon.github.io/docs/installing/fdo/ (missing a spacer between link and glyph)
-  https://open-horizon.github.io/docs/integrating_index/ (broken link)

## Checklist:

- [x] I have tagged the reviewers in a comment below incase my pull request is ready for a review
- [x] I have signed the commit message to agree to Developer Certificate of Origin (DCO) (to certify that you wrote or otherwise have the right to submit your contribution to the project.) by adding "--signoff" to my git commit command.

### CODE REVIEW

File: docs/installing/fdo.md [hunk 1]
```diff
@@ -59,7 +59,7 @@ Before you purchase FDO-enabled edge devices, you can test FDO support in {{site
    ```
    {: codeblock}
 
-3. Follow the steps in the [FIDO Device Onboard (FDO)](https://github.com/open-horizon/FDO-support/blob/main/README.md){:target="_blank"}{: .externalLink} to observe FDO automatically install the {{site.data.keyword.ieam}} agent on a device and registers it with your {{site.data.keyword.ieam}} management hub.
+3. Follow the steps in the [FIDO Device Onboard (FDO) ](https://github.com/open-horizon/FDO-support/blob/main/README.md){:target="_blank"}{: .externalLink} to observe FDO automatically install the {{site.data.keyword.ieam}} agent on a device and registers it with your {{site.data.keyword.ieam}} management hub.
```
Comment (joewxboy) [inline docs/installing/fdo.md:62]: Should we link to this document instead?
https://open-horizon.github.io/docs/fdo/README-FDO1.1/

**Review by joewxboy — APPROVED** [review]

File: docs/installing/fdo.md (hunk 1 above)
Comment (johnwalicki) [reply docs/installing/fdo.md:62]: The "FIDO Device Onboard (FDO) Protocol Reference Implementation (PRI) Quick Start" is relatively low level.
The current link goes to the FDO README.  This commit just fixes how the External Link glyph and text looks like.

There is longer term discussion on what the FDO landing page and documents should be.  See #439 
At that point, this particular page will be modified.  Suggest we defer the FDO decision to #439

**Comment by johnwalicki** [comment]
Fixes #448 as well.

**Comment by johnwalicki** [comment]
After this PR was merged, I confirmed that the external links are now all correct.

**Pull request merged**